DataFormats/common/include/CommonDataFormat/InteractionRecord.h

```diff
@@ -117,6 +117,13 @@ struct InteractionRecord {
     return (int64_t(orbit) * o2::constants::lhc::LHCMaxBunches) + bc;
   }
 
+  // get global BC ID from bc and orbit
+  // this ID is needed to calculate BC rel. to BCID of vertex
+  uint64_t getGlobalBC() const
+  {
+    return bc + orbit * o2::constants::lhc::LHCMaxBunches;
+  }
```

Comment:
this functions seems introduced but not used in the code anywhere. Do you need it?

Reply:
note that there is helper function ```uint64_t relativeTime_to_GlobalBC(double relativeTimeStampInNS)
``` in the AOD converter itself which may be interesting!?

Reply:
the function was not needed  and an artifact from previous implementations. I removed it